plugins/modules/files/iso_customize.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: iso_customize
+short_description: Customize the ISO file by add/delete/modify files
```

Comment:
How about:
```suggestion
short_description: Add/remove/change files in ISO files
```

Reply:
Fix it. Thanks for your careful review.

Reply:
@ZouYuhua you are marking the comments as "resolved" before actually resolving them. That is quite hard to keep track of.

I suggest you go the other way around: fix and push the fix first, then you come back and "resolve" them.

Reply:
Thanks very much. 
Process: fix and push the fix at first, pass the tests, then mark as resolved.